CRM/Mailing/BAO/Mailing.php

```diff
@@ -1469,7 +1469,7 @@ private function getTokenData(&$token_a, $html = FALSE, &$contact, &$verp, &$url
     if ($type == 'embedded_url') {
       $embed_data = array();
       foreach ($token as $t) {
-        $embed_data[] = $this->getTokenData($t, $html = FALSE, $contact, $verp, $urls, $event_queue_id);
+        $embed_data[] = $this->getTokenData($t, $html, $contact, $verp, $urls, $event_queue_id);
```

Comment:
This was always passing in true (because it was setting it to FALSE) - my brain hurts :P